src/awesome-texmacs.tm

```diff
@@ -1,11 +1,13 @@
 <TeXmacs|1.99.14>
 
-<style|notes>
+<style|browser>
 
 <\body>
-  <hlink|[index]|./index.tm><chapter*|Awesome GNU TeXmacs>
+  <hlink|[index]|./index.html><label|auto-1>
```

Comment:
Also here. Please leave the link to .tm, the conversion is done automatically later on.